packages/app-project/src/components/AuthModal/AuthModal.js

```diff
@@ -42,6 +44,11 @@ function AuthModal (props) {
             <LoginForm closeModal={closeModal} />
           </Box>
         </Tab>
+        <Tab title='Register'>
+          <Box width='large'>
```

Comment:
I've done this for LoginModal as well, but this `Box` is unnecessary, the width prop can be set inside RegisterModal